src/main/webapp/person/visualizePersonalInfo.jsp

```diff
@@ -64,7 +64,7 @@
 			<tr>
 				<td align="center">
 					<div class="items-container" data-toggle="tooltip" data-placement="right" title="${fr:message('resources.ApplicationResources', 'link.operator.submitPhoto')}">
-						<img src="${fr:checksum('/person/retrievePersonalPhoto.do?method=retrieveOwnPhoto')}" width="100" height="100" align="middle" style="border: 2px #eee solid"/>
+						<img src="${fr:checksum('/person/retrievePersonalPhoto.do?method=retrieveOwnPhoto&'.concat(person.personalPhotoEvenIfPending.externalId))}" width="100" height="100" align="middle" style="border: 2px #eee solid"/>
```

Comment:
In this invocation, the id is just included for Cache Busting, it is not actually used.